doc/setup_instructions.md

```diff
@@ -84,12 +84,23 @@ desktop development featureset, that should have everything needed for
 C# development. Please be patient with this as this is going to take
 tens of minutes to install and take a lot of disk space.
 
-It might also be possible to make things work with Visual studio code
-as well, but there are no instructions for that. Note: that Godot
-requirements say that visual studio build tools are enough but they
-may not include nuget, which is needed, so if you go that route you
-may need to manually install nuget.
+It is also be possible to make things work with visual studio code
+as well, but the setup process is more complicated and more likely to not work. Note: that Godot requirements say that visual studio build tools are enough but they may not include nuget, which is needed, so if you go that route you may need to manually install nuget.
```

Comment:
Did you change anything here? To me it looks like you just made the text lines just longer.

Reply:
I don't remember changing anything there